src/components/Hero.tsx

```
@@ -43,26 +48,45 @@
             <span>Featured on Hacker News</span>
           </a>
           <div
-            className="scale-75 md:scale-90"
+            className="sm:scale-75"
             dangerouslySetInnerHTML={{ __html: productHuntBadgeHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)